xray/utils/resultstable.go

```diff
@@ -258,8 +288,8 @@ func splitComponentId(componentId string) (string, string, string) {
 }
```

Comment:
Let's add a description for the ```splitComponentId(componentId``` function, specifically describing the three returned values. I also suggest adding a few examples to the description.